source/base/data_out_base.cc

```diff
@@ -5675,8 +5674,8 @@ namespace DataOutBase
                                     /* use VTU, not VTK: */ false);
                               local_vertex_order[connectivity_index] =
                                 local_index;
-                              flush_current_cell();
                             }
+                          flush_current_cell();
```

Comment:
This doesn't seem right, looking at where we call `flush_current_cell` in the other cases. There, we do it for every subdivision, not for every cell. The old code looked correct to me.

Reply:
I think you are talking about other usages of this function. In the case that we do not use the high-order functionality, this is indeed the case:

https://github.com/dealii/dealii/blob/ba82c4215fa57c8e2560d4b23ead55cd1ae2ddec/source/base/data_out_base.cc#L5570-L5642

But in the high-order case, we output once per cell:

https://github.com/dealii/dealii/blob/ba82c4215fa57c8e2560d4b23ead55cd1ae2ddec/source/base/data_out_base.cc#L5663-L5731

Reply:
Oh, I see. Thanks for checking!